tests/no-learning/test_DDM.py

```diff
@@ -61,3 +64,30 @@ def test_DDM():
         # if you do not specify, assert_allcose will use a relative tolerance of 1e-07,
         # which WILL FAIL unless you gather higher precision values to use as reference
         np.testing.assert_allclose(val, expected, atol=1e-08, err_msg='Failed on expected_output[{0}]'.format(i))
+
+class TestDDMCorrectness:
```

Comment:
no need for the class, use just a plain function.
Also please put this in `tests/mechanisms/test_ddm_mechanism.py`, this file only has tests using deprecated system/processes.